### PR TITLE
enforce static imports

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/internal/OperationMetricsUtilTest.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 package io.opentelemetry.instrumentation.api.internal;
 
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
```
Comment (trask) [inline instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/internal/OperationMetricsUtilTest.java:8]: looks like no need for OpenTelemetryAssertions in this class
```suggestion
static import org.assertj.core.api.Assertions.assertThat;
```

File: instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/LogReplayOpenTelemetryAppenderTest.java [hunk 2]
```diff
@@ -44,8 +44,8 @@ void twoLogs() {
     OpenTelemetryAppender.install(openTelemetry);
 
     List<LogRecordData> logDataList = logRecordExporter.getFinishedLogRecordItems();
-    assertThat(logDataList).hasSize(1);
-    OpenTelemetryAssertions.assertThat(logDataList.get(0))
+    Assertions.assertThat(logDataList).hasSize(1);
```
Comment (trask) [inline instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/LogReplayOpenTelemetryAppenderTest.java:47]: does this work?
```suggestion
    assertThat(logDataList).hasSize(1);
```

File: instrumentation/opentelemetry-api/opentelemetry-api-1.31/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/v1_31/metrics/MeterTest.java [hunk 3]
```diff
@@ -108,7 +108,7 @@ void longCounter() {
   void doubleCounter() {
     DoubleCounterBuilder builder =
         meter.counterBuilder("test").ofDoubles().setDescription("d").setUnit("u");
-    assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
+    Assertions.assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
```
Comment (trask) [inline instrumentation/opentelemetry-api/opentelemetry-api-1.31/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/v1_31/metrics/MeterTest.java:111]: (same question in a few places)
```suggestion
    assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
```

File: instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/LogReplayOpenTelemetryAppenderTest.java (hunk 2 above)
Comment (zeitlinger) [reply instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/LogReplayOpenTelemetryAppenderTest.java:47]: yes, this should also be disallowed

Comment (zeitlinger) [reply instrumentation/log4j/log4j-appender-2.17/library/src/test/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/LogReplayOpenTelemetryAppenderTest.java]: is disallowed now

**Review by trask — APPROVED** [review]
nice

**Pull request merged**